lightblue/src/test/java/org/esbtools/eventhandler/lightblue/BulkLightblueRequesterTest.java

```diff
@@ -433,7 +439,43 @@ public void shouldCacheBothGuaranteedRequestsAndTriedRequestsTogetherAndSendToLi
             throw e;
         }
     }
-
+    
+    @Test
+    public void shouldBeOrderedIfRequested() throws Exception {
+
+        LightblueClient mockClient = Mockito.mock(LightblueClient.class);
+        LightblueBulkDataResponse lightblueDataResponse = Mockito.mock(LightblueBulkDataResponse.class);
+        when(mockClient.bulkData(any(DataBulkRequest.class))).thenReturn(lightblueDataResponse);
+
+        BulkLightblueRequester orderedRequester = new BulkLightblueRequester(mockClient, true);
+        orderedRequester.request(new DataFindRequest("foo"), new DataFindRequest("bar")).get();
+        Mockito.verify(mockClient).bulkData(Mockito.argThat(new ArgumentMatcher<DataBulkRequest>() {
+            @Override
+            public boolean matches(Object argument) {
+                assertTrue(((DataBulkRequest) argument).isOrdered());
+                return true;
+            }
+        }));
+    }
+    
+    @Test
+    public void shouldBeUnOrderedIfRequested() throws Exception {
+
+        LightblueClient mockClient = Mockito.mock(LightblueClient.class);
+        LightblueBulkDataResponse lightblueDataResponse = Mockito.mock(LightblueBulkDataResponse.class);
+        when(mockClient.bulkData(any(DataBulkRequest.class))).thenReturn(lightblueDataResponse);
+
+        BulkLightblueRequester orderedRequester = new BulkLightblueRequester(mockClient, false);
+        orderedRequester.request(new DataFindRequest("foo"), new DataFindRequest("bar")).get();
+        Mockito.verify(mockClient).bulkData(Mockito.argThat(new ArgumentMatcher<DataBulkRequest>() {
+            @Override
+            public boolean matches(Object argument) {
+                assertFalse(((DataBulkRequest) argument).isOrdered());
```

Comment:
Same deal here, we want to return true if the request "matches" our expectation, which is that the request should be unordered in this case.